src/control/server/ctl_storage_rpc_test.go

```diff
@@ -575,7 +594,10 @@ func TestServer_CtlSvc_StorageScan_PostIOStart(t *testing.T) {
 			},
 		},
 		"scan bdev meta with multiple io servers up with multiple nvme namespaces": {
-			req: &StorageScanReq{Nvme: &ScanNvmeReq{Meta: true}},
+			req: &StorageScanReq{
+				Scm:  new(ScanScmReq),
```

Comment:
Do we still behave correctly if these are nil? since they're pointers

Reply:
The sub functions will refuse nil, the references are always non-nil as they get populated in the control API function that calls StorageScan (StorageScan handler is only ever called through the control API).

Reply:
Cool, got it.